src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -12397,14 +12397,15 @@ void func_8084ECA4(Player* this, PlayState* play) {
     if (LinkAnimation_Update(play, &this->skelAnime)) {
         if (this->unk_84F != 0) {
             if (this->unk_850 == 0) {
-                Message_StartTextbox(play, D_80854A04[this->unk_84F - 1].textId, &this->actor);
+                Message_StartTextbox(play, sBottleCatchInfo[this->unk_84F - 1].textId, &this->actor);
                 Audio_PlayFanfare(NA_BGM_ITEM_GET | 0x900);
                 this->unk_850 = 1;
             } else if (Message_GetState(&play->msgCtx) == TEXT_STATE_CLOSING) {
                 this->unk_84F = 0;
                 func_8005B1A4(Play_GetCamera(play, CAM_ID_MAIN));
             }
         } else {
+            // Changes the animation, which causes the bug at the bottom of the function.
             func_8083C0E8(this, play);
         }
```

Comment:
Id say this comment is probably not needed, only cause it will be clear that it changes the animation when its named in the future and the bug comment at the bottom refers directly to it

Reply:
I thought it was worth it at least for now, but I can remove it.